### ISSUE TITLE
[NTI] WARNING - All constants must be typed

### ISSUE THREAD

**Issue by vobruba-martin** [google/closure-compiler#2033]
@dimvar I have a repro you wanted [here](https://github.com/google/closure-compiler/commit/05fa13a146ea9f9570994b04e7974b6d58283d37#commitcomment-19057351)

I run compiler with these args:
```
--js test.js
--externs externs.js
--new_type_inf
--compilation_level ADVANCED
--warning_level VERBOSE
```

`test.js` is an empty file

`externs.js`:
```javascript
/**
 * @externs
 */

/** @constructor */
function jQuery() {};

/** @const */
var $ = jQuery;
```

And I get this warning:
```
externs.js:16: WARNING - All constants must be typed. The compiler could not infer the type of constant window.$. Please use an explicit type annotation. For more information, see:
https://github.com/google/closure-compiler/wiki/Using-NTI-(new-type-inference)#warnings-about-uninferred-constants
var $ = jQuery;
    ^

0 error(s), 1 warning(s)
```

**Comment by dimvar** [comment]
Looks like the aliasing is supported only in the actual code, not in the externs. I'll fix this.

**Comment by Dominator008** [comment]
Fixed by 73c6a9aa3de9ca25e88f99287355316aa8cf3120.
